compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/ConstExpr.cpp

```diff
@@ -21,6 +22,12 @@ namespace iree_compiler {
 namespace IREE {
 namespace Util {
 
+static llvm::cl::opt<int64_t> clConstExprMaxSizeIncreaseThreshold(
```

Comment:
We will likely want to plumb this through as a real option vs a global cl option (so it can be controlled by API). Fine to do later.

Reply:
Filed an issue for this to do later: https://github.com/openxla/iree/issues/15073